drakrun/drakrun/draksetup.py

```diff
@@ -136,11 +149,16 @@ def install(vcpus, memory, storage_backend, disk_size, iso_path, zfs_tank_name,
 
     sha256_hash = hashlib.sha256()
 
-    with open(iso_path, "rb") as f:
-        for byte_block in iter(lambda: f.read(4096), b""):
-            sha256_hash.update(byte_block)
+    logging.info("Calculating hash of iso")
+    iso_file_size = os.stat(iso_path).st_size
+    block_size = 128 * 1024
+    with tqdm(total=math.ceil(iso_file_size), unit_scale=True) as pbar:
```

Comment:
We can now pass `total=iso_file_size` (since it's just noop at this point) and drop `import math`.

Apart from that LGTM

Reply:
Done